src/telcameras_v2/serializers.py

```diff
@@ -16,7 +16,7 @@ class Meta:
             'azimuth',
             'count_in',
             'count_out',
-            'area',
```

Comment:
Deze dingen worden ook in aggregaties gebruikt, zoals bv:
- https://github.com/Amsterdam/waarnemingen-mensen/blob/master/src/telcameras_v2/view_definitions.py#L4946-L4956
- https://github.com/Amsterdam/waarnemingen-mensen/blob/master/src/continuousaggregate/view_definitions.py#L99-L100
- https://github.com/Amsterdam/waarnemingen-mensen/blob/master/src/continuousaggregate/view_definitions.py#L122-L123
- https://github.com/Amsterdam/waarnemingen-mensen/blob/master/src/continuousaggregate/view_definitions.py#L140-L150